api/src/certification/session/domain/usecases/upload-cpf-files.js

```diff
@@ -0,0 +1,19 @@
+import { config } from '../../../../../lib/config.js';
+import { S3ObjectStorageProvider } from '../../../../shared/storage/infrastructure/providers/S3ObjectStorageProvider.js';
+const { cpf } = config;
+
+const uploadCpfFiles = async function ({
+  filename,
+  readableStream,
+  logger,
+  dependencies = { S3ObjectStorageProvider },
+}) {
+  logger.trace('uploadCpfFiles: start upload');
+  await dependencies.S3ObjectStorageProvider.createClient({ ...cpf.storage, dependencies: { logger } }).startUpload({
+    filename,
+    readableStream,
+  });
```

Comment:
juste histoire d'être iso dans l'écriture avec get-cpf-presigned-urls

```suggestion
 const S3ObjectStorageProvider = await dependencies.S3ObjectStorageProvider.createClient({ ...cpf.storage, dependencies: { logger } });
 
 await S3ObjectStorageProvider.startUpload({
    filename,
    readableStream,
  });
```

Reply:
Done